validator/client/runner.go

```diff
@@ -39,11 +38,6 @@ func run(ctx context.Context, v iface.Validator) {
 		cleanup()
 		log.Fatalf("Wallet is not ready: %v", err)
 	}
-	if featureconfig.Get().SlasherProtection {
```

Comment:
Slasher is now part of the beacon node, there is no need for a "SlasherReady" function